src/alhambra/mixes.py

```diff
@@ -43,8 +43,9 @@
 
 import warnings
 
-from pint.quantity import Quantity
+import scadnano as sc
```

Comment:
If we need scadnano (and it looks like a non-release version?), could you add the right url/version requirement to setup.cfg?

Reply:
Or could these use WellPos?

Reply:
> If we need scadnano (and it looks like a non-release version?), could you add the right url/version requirement to setup.cfg?

I was just being lazy, and I thought for some reason scadnano was already a dependency. I can change it so it doesn't depend on scadnano.